ta4j-core/src/test/java/org/ta4j/core/analysis/criteria/BuyAndHoldCriterionTest.java

```diff
@@ -56,29 +56,29 @@ public void calculateOnlyWithLossTrades() {
                 Order.buyAt(2, series), Order.sellAt(5, series));
 
         AnalysisCriterion buyAndHold = new BuyAndHoldCriterion();
-        assertEquals(0.7, buyAndHold.calculate(series, tradingRecord), TestUtils.GENERAL_OFFSET);
+        assertNumEquals(0.7, buyAndHold.calculate(series, tradingRecord));
     }
 
     @Test
     public void calculateWithNoTrades() {
         MockTimeSeries series = new MockTimeSeries(numFunction, 100, 95, 100, 80, 85, 70);
 
         AnalysisCriterion buyAndHold = new BuyAndHoldCriterion();
-        assertEquals(0.7, buyAndHold.calculate(series, new BaseTradingRecord()), TestUtils.GENERAL_OFFSET);
+        assertNumEquals(0.7, buyAndHold.calculate(series, new BaseTradingRecord()));
     }
     
     @Test
     public void calculateWithOneTrade() {
         MockTimeSeries series = new MockTimeSeries(numFunction, 100, 105);
         Trade trade = new Trade(Order.buyAt(0, series), Order.sellAt(1, series));
         AnalysisCriterion buyAndHold = new BuyAndHoldCriterion();
-        assertEquals(105d/100, buyAndHold.calculate(series, trade), TestUtils.GENERAL_OFFSET);
+        assertNumEquals(105d/100, buyAndHold.calculate(series, trade));
     }
 
     @Test
     public void betterThan() {
         AnalysisCriterion criterion = new BuyAndHoldCriterion();
-        assertTrue(criterion.betterThan(1.3, 1.1));
-        assertFalse(criterion.betterThan(0.6, 0.9));
+        assertTrue(criterion.betterThan(numOf(1.3), numOf(1.1)));
+        assertFalse(criterion.betterThan(numOf(0.6), numOf(0.9)));
     }
 }
```

Comment:
BuyAndHold Test looks good.  Am wondering if `double` values like `0.7` should be replaced with `Nums`.